src/pipeline.ml

```diff
@@ -568,7 +614,7 @@ module Mirage = struct
       ];
     ]
 
-  let v ?app ?notify:channel ~sched ~staging_auth () =
+  let v ?app ?notify:channel ?filter:_ ~sched ~staging_auth () =
```

Comment:
The ignored `filter` option allows us to have a uniform interface for all the deployer modules.